package.json

```diff
@@ -76,5 +81,6 @@
     ],
     "coverageDirectory": "../coverage",
     "testEnvironment": "node"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
is it necessary to specify the hash here?

Reply:
Those lines are generated by `npm install`.
I'd rather remove yarn if you prefer.
Yan is an abandoned project. It's not supported anymore and not included in node distribution now.

Reply:
removed